src/Configuration/NodeDefinition/TableNodesDecorator.php

```diff
@@ -78,6 +65,7 @@ public function addNodes(NodeBuilder $builder): void
         $this->addTableNode($builder);
         $this->addColumnsNode($builder);
         $this->addOutputTableNode($builder);
+        $this->addIncrementalLoading($builder);
```

Comment:
V `TableNodesDecorator` bola pridana metoda `addIncrementalLoading` ... kedze pred tym to bola chybne sucast `addIncrementalFetchingNodes`.